lib/cb/clients/recommendation.rb

```diff
@@ -69,7 +69,9 @@ def self.for_company(company_did)
         if json_hash
           api_jobs = json_hash.fetch('Results', {}).fetch('JobRecommendation', {}).fetch('Jobs', {})
           if api_jobs
-            api_jobs.fetch('CompanyJob', []).each do |cur_job|
+            company_jobs = api_jobs.fetch('CompanyJob', [])
+            company_jobs = [company_jobs] unless company_jobs.is_a?(Array)
+            company_jobs.each do |cur_job|
```

Comment:
You can also just use `Array(api_jobs.fetch)` which will wrap it if it's not an array, and leave it alone if it is.